web/client/reducers/layers.js

```diff
@@ -248,6 +249,24 @@ function layers(state = { flat: [] }, action) {
             groups: newGroups
         });
     }
+    case UPDATE_LAYER: {
```

Comment:
Why do we need this action? there is already updateNode or addNewLayer, could we use that instead?

Reply:
Removed action and reverted to the already implemented addNewLayer here https://github.com/geosolutions-it/MapStore2/pull/10293/commits/98f88d219a9dd148be8abdc52d713616a5820935